docs/sources/alerting/configure-notifications/manage-contact-points/integrations/webhook-notifier.md

```diff
@@ -24,9 +24,11 @@ title: Configure the webhook notifier for Alerting
 weight: 200
 ---
 
-### Configure the webhook notifier for Alerting
+## Configure the webhook notifier for Alerting
```

Comment:
this should be header 1 #

Reply:
oh then we did not get it right in the skack and telegram docs 😁